src/v2/System/SystemContext.tsx

```diff
@@ -77,6 +77,9 @@ export interface SystemContextProps extends SystemContextState {
 
   /** Is there a logged in user? */
   isLoggedIn?: boolean
+
+  /** FeatureFlags */
+  featureFlags?: Record<string, boolean>
```

Comment:
@mc-jones - didn't catch this before merge, but one tiny little nbd follow up is this could use the `EnabledFeatureFlags` type that was added below and shared with `Boot`